app/utils/build-provider-asset-path.js

```diff
@@ -0,0 +1,15 @@
+import config from 'ember-get-config';
+import pathJoin from '../utils/path-join';
+
+export default function buildProviderAssetPath(providerId, assetName, isDomain) {
```

Comment:
If you passed in the config.providerAssetsURL rather than pulling it here, then it would be super-simple to test.